man/as.matrix.Rd

```diff
@@ -7,7 +7,7 @@ Converts a \code{data.table} into a \code{matrix}, optionally using one
 of the columns in the \code{data.table} as the \code{matrix} \code{rownames}.
 }
 \usage{
-\method{as.matrix}{data.table}(x, rownames, ...)}
+\method{as.matrix}{data.table}(x, rownames, rownames.literal, ...)}
```

Comment:
This differ from function code, default value, and should raise a warning when checking package

Reply:
it didn't, maybe methods have less checks, still should be corrected to include default value, as in function definition. Also instead of `...` use `\dots` keyword.

Reply:
Thanks - i'm surprised the checks didnt pick this up (even `R CMD check --as-cran` misses this).  

As an aside, is there any reason we don't use Roxygen to generate documentation? This would avoid a lot of these fiddly issues arising from manually creating the .Rd files.

Reply:
I don't remember exact reasoning when we were discussing roxygen. As for me, as of now (I used to use roxygen everywhere), I don't like to "touch" source code when I only need to edit documentation. Model of keeping documentation in source files brings a lot of "noise" in git history. Not even mentioning stuff like [this](https://github.com/klutometis/roxygen/pull/334), because of which I had to use my own fork of roxygen, till RStudio stops to support such "old" version of roxygen. Then I realized the chain of dependencies of my development helpers was not actually helping as much as disturbing :)